src/ops/ops.go

```diff
@@ -22,10 +23,19 @@ import (
 	"github.com/openshift/assisted-installer/src/utils"
 )
 
+type CmdContext struct {
+	Out io.Writer
+	Env []string
+	Dir string
+}
+
 //go:generate mockgen -source=ops.go -package=ops -destination=mock_ops.go
 type Ops interface {
 	ExecPrivilegeCommand(liveLogger io.Writer, command string, args ...string) (string, error)
 	ExecCommand(liveLogger io.Writer, command string, args ...string) (string, error)
+	ExecPrivilegeCommandWithContext(cmdctx *CmdContext, command string, args ...string) (string, error)
```

Comment:
I think we need to move all to use context and not to add new functions

Reply:
I have a separate PR to handle this. It a huge change and needed to be committed separately